src/pages/Overview/index.tsx

```diff
@@ -86,7 +86,7 @@ const OverviewCommon = ({ allAuctions }: OverviewProps) => {
         'No'
       ),
       selling:
-        item.symbolAuctioningToken == 'WXDAI' ? 'XDAI' : item.symbolAuctioningToken.slice(0, 7),
+        item.symbolAuctioningToken == 'WXDAI' ? 'XDAI' : item.symbolAuctioningToken.slice(0, 12),
```

Comment:
this doesn't matter at all actually but I was just changing all token symbol .slice(0, 7) entries